<launch>

	<arg name="use_sim_time" default="true"/>
	<arg name="gui" default="true"/>
	<arg name="headless" default="false"/>
	<arg name="debug" default="false"/>

	<include file="$(find plen_ros)/launch/gazebo_plen.launch">
		<arg name="debug" value="$(arg debug)" />
	    <arg name="gui" value="$(arg gui)" />
	    <arg name="use_sim_time" value="$(arg use_sim_time)"/>
	    <arg name="headless" value="$(arg headless)"/>
	</include>

	<node name="gazebo_tools_test" pkg="plen_ros" type="gazebo_tools_test"/>


	<include file="$(find plen_ros)/launch/traj_controller.launch"/>

	<arg name="td3" default="False"/>
	<group if="$(arg td3)">
	  <node name="plen_td3" pkg="plen_ros" type="plen_td3"/>
	</group>
</launch>